recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -132,20 +132,27 @@ def package(self):
         # remove unneeded directories
         tools.rmdir(os.path.join(self.package_folder, "share"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "doc"))
         # remove binaries and pdb files
         for pattern_to_remove in ["cjpeg*", "djpeg*", "jpegtran*", "tjbench*", "wrjpgcom*", "rdjpgcom*", "*.pdb"]:
             for bin_file in glob.glob(os.path.join(self.package_folder, "bin", pattern_to_remove)):
                 os.remove(bin_file)
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "libjpeg-turbo"
+        self.cpp_info.names["cmake_find_package_multi"] = "libjpeg-turbo"
+
+        cmake_target_suffix = "-static" if not self.options.shared else ""
+        lib_suffix = "-static" if self.settings.compiler == "Visual Studio" and not self.options.shared else ""
+
+        self.cpp_info.components["jpeg"].names["cmake_find_package"] = "jpeg" + cmake_target_suffix
```

Comment:
I fear (some) packages breaking that depends on libjpeg-turbo.

Reply:
Do you mean `-static` suffix?

Reply:
This is why it's always dangerous to rely on non official CMake targets. When there is no official CMake target, I prefer to inject CONAN_LIBS or CONAN_PKG targets in recipes, because there can't break in new revisions.

Reply:
It's okay we break recipes all the time in subtle ways... its unavoidable since we are always leading with new features

Reply:
Yes, but it's important to check at least if it can break others CCI recipes. Here I think it's ok.